src/click.ts

```diff
@@ -12,38 +12,45 @@ export async function click (context: BrowserContext, userInput: string, keyword
       const pages = context.pages()
       const page = pages[pages.length - 1]
       const elements = await page.locator(`${locator}`).all()
+
+      // Look in the iframes
+      const frames = page.frames()
+      for (const f of frames) {
+        if (f.url() !== '' && f.url() !== 'about:blank') {
+          elements.push(...await f.locator(`${locator}`).all())
+        }
+      }
+
       if (elements.length === 0) {
         console.log(`No elements found for locator: ${locator}`)
         continue
       }
       for (const element of elements) {
-        await element.scrollIntoViewIfNeeded()
-        const innerText = await getText(element)
-        for (const keyword of keywords) {
-          if (innerText.toLowerCase().includes(keyword.toLowerCase())) {
```

Comment:
I could not find any compelling reason to ensure that the keywords were present in the innerText of the element to be clicked on (since the elements were already selected by a call to the LLM anyway), so I removed this check.